joomla/joomla-cms
# [5.2] TinyMCe insert content

This changes the button text used when selecting a content template from "Save" to "Insert".

No additional changes are required for this to be translated as "Insert" is already translated.

(This PR was created at the request of "Daniel C K Tan" on facebook.

### Testing Instructions
If you dont already have content templates set up and working for tinymce go to the plugin settings for tinymce and setup the correct folder
![image](https://github.com/joomla/joomla-cms/assets/1296369/5d5c88fc-d2e7-4f3d-b6ce-c3739cd0ab6f)

Then create a .txt file in that folder. (As this is just to test the UI the file can be empty)

Finaly in an article select the content template icon from the toolbar
![image](https://github.com/joomla/joomla-cms/assets/1296369/1dcad79e-f5ad-4ccf-989a-df614a5eb2c9)

And in the dialog that opens observe the text of the buttons.

Then apply this PR and retest. As it is a js change either apply the pr and then run `npm i` or use the prebuilt joomla package that can be found at the bottom of this page at the download link

### Actual result BEFORE applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1296369/69f1de31-ae53-41cf-a54a-e372cae42ca7)



### Expected result AFTER applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/1296369/edb452de-1b42-4f25-8bb0-8e1e0d1ddd0c)



### Link to documentations
Please select:
